force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -40,17 +40,23 @@ public with sharing class ClassesObjectsMethods {
      */
     public static List<String> practiceStringMethods() {
         // Initialize a String with 'Hello, Salesforce!'
-        
-        // Get the length of the String        
+        String myString = 'Hello, Salesforce!';
+        // Get the length of the String
+        integer length = myString.length();
         // Get a substring of the String of the first 5 characters
-        // Convert the String to lower case
+        String substring = myString.substring(0, 5);
+        // Get the string in lower case
+        String lowerCase = myString.toLowerCase();
         
         // Return the list with the string data
+        List<String> resultList = new List<String>();
         // In the first position add the string length
+        resultList.add(String.valueOf(length));
         // In the second position add the substring
+        resultList.add(substring);
         // In the third position add the string in lower case
-
-        return null; // Replace null with the variable you used to store the result
+        resultList.add(lowerCase);
+        return resultList; // Replace null with the variable you used to store the result
```

Comment:
Good work. You could just add the  result of your our calls to the String methods directly to the list. 

```
 strings.add(String.valueOf(helloSalesforce.length()));
        strings.add(helloSalesforce.substring(0, 5));
        strings.add(helloSalesforce.toLowerCase());
        return strings;
        ```